package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -6,37 +6,37 @@ function restart_tunneldigger()
 end
 
 function read_pid_file()
-	local pidFile = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
-	if not pidFile then
+	local pid_file = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	if not pid_file then
 		return nil
 	end
-	returnPid = pidFile:read("*l") 
-	io.close(pidFile)
-	return returnPid
+	pid = pid_file:read("*l") 
+	pid_file:close()
+	return pid
 end
 
 function check_for_mesh_vpn_neighbours()
-	local handleBatctl = io.popen("batctl o", 'r')
-	if handleBatctl~=nil then
-		for line in handleBatctl:lines() do
+	local handle_batctl = io.popen("batctl o", 'r')
+	if handle_batctl~=nil then
```

Comment:
if not handle_batctl then, in fact I would call the variable something like `response`.